js/bzconfig.json

```diff
@@ -0,0 +1,43 @@
+{
+  "BUGZILLA_URL": "https://bugzilla.mozilla.org/buglist.cgi",
+  "BUGZILLA_REST_URL": "https://bugzilla.mozilla.org/rest/bug",
+  "versions": {
+    "release": {
+      "version": 45,
+      "title": "Firefox",
+      "img": "images/firefox.png"
+    },
+	"beta": {
```

Comment:
Hm yeah I have a mix of spaces and tabs. I'm used to spaces, but the JS here is tab-indented, so apparently I mixed up the two in the new file. To match the original style, I'll tab-indent it.